### PR TITLE
Thomas/cls model/v1

### PR DESCRIPTION
**Pull request by thomasfortin1**

### CODE REVIEW

File: models/preprocessing.py [hunk 1]
```diff
@@ -87,7 +124,10 @@ def glob_to_snapshot(top_level_path):
 
     # Labels
     prefix = top_level_path + "/Labels/"
-    files = glob.glob(prefix + "*/*")
+    if extra_level:
+        files = glob.glob(prefix + "Mimic3/*/*")
```
Comment (trevor-yu-087) [inline models/preprocessing.py:128]: Ah if you just put in Mimic3 in the top level path it'll work. oh well not a huge deal.

File: models/preprocessing.py [hunk 2]
```diff
@@ -11,6 +11,40 @@
 from torch.utils.data import Dataset, DataLoader
 import torch
 
+HEAD = 0
```
Comment (trevor-yu-087) [inline models/preprocessing.py:14]: Just make the dict with the label values right in the values instead of declaring all these CHEST/MRI vars?

File: models/ner_model.py [hunk 3]
```diff
@@ -1,10 +1,12 @@
-from .training_model_api import TrainingModel
```
Comment (trevor-yu-087) [inline models/ner_model.py:1]: These should stay as relative imports I think? Because when you import NerModel from gui_v0 in a notebook, you'd want these to work. 
e.g.
`from gui_v0.models.ner_model import NerModel` won't work if it says `from models import ...`
I might be wrong but it didn't work when I tried it in jupyter in the server.

File: models/cls_model.py [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+                eval_preds.extend(predictions)
+
+                tmp_eval_accuracy = accuracy_score(labels.cpu().numpy(), predictions.cpu().numpy())
+                eval_accuracy += tmp_eval_accuracy
+
+        labels = [id.item() for id in eval_labels]
+        predictions = [id.item() for id in eval_preds]
+
+        cm = confusion_matrix(labels, predictions)
+        # Get accuracy report of just tags
```
Comment (trevor-yu-087) [inline models/cls_model.py:237]: Remove the comments

File: models/cls_model.py [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+            data snapshot for training/validation/test data
+        generate_labels : bool, optional
+            Whether to generate training labels (for training and validation), by default True
+
+        Returns
+        -------
+        pd.DataFrame
+            Data transformed into dataframe with keys for ['text', 'id'] and ['label'] if labels are generated
+        """
+        # if generate_labels:
```
Comment (trevor-yu-087) [inline models/cls_model.py:63]: Keep the logic for generate labels because if text is passed without labels present (e.g. in inference) then we should do some other preprocessing (likely just text splitting).

**Review by trevor-yu-087 — CHANGES_REQUESTED** [review]

File: models/ner_model.py (hunk 3 above)
Comment (thomasfortin1) [reply models/ner_model.py:1]: Oh weird. The relative imports weren't working for me when I was developing locally or on the server so I had to write models.<<whatever>>. Maybe we could just do absolute from gui_v0 everywhere?
